testcases/parser/nodes.go

```diff
@@ -59,6 +59,47 @@ func (tc *TestCase) GetFunctionOptions() []*types.FunctionOption {
 	return funcOptions
 }
 
+func (tc *TestCase) getScalarFuncArgTypes() []types.Type {
+	argTypes := make([]types.Type, len(tc.Args))
+	for i, arg := range tc.Args {
+		argTypes[i] = arg.Type
+	}
+	return argTypes
+}
+
+func (tc *TestCase) getAggregateFuncArgTypes() []types.Type {
+	argTypes := make([]types.Type, len(tc.AggregateArgs))
```

Comment:
I've always wondered -- given that both Args and AggregateArgs have the same type why do we have both if only one is populated at a time?